src/main/java/Alpha/Alpha.java

```diff
@@ -0,0 +1,250 @@
+        sendMessage("Uh oh! I'm not sure how to react to that, have an ice cream instead!"
+            + System.lineSeparator()
+            + "         _.-." + System.lineSeparator()
+            + "       ,'/ //\\" + System.lineSeparator()
+            + "      /// // /)" + System.lineSeparator()
+            + "     /// // //|" + System.lineSeparator()
+            + "    /// // ///" + System.lineSeparator()
+            + "   /// // ///" + System.lineSeparator()
+            + "  (`: // ///" + System.lineSeparator()
+            + "   `;`: ///" + System.lineSeparator()
+            + "   / /:`:/" + System.lineSeparator()
+            + "  / /  `'" + System.lineSeparator()
+            + " / /" + System.lineSeparator()
+            + "(_/");
+    }
+
+    private static void writeTasksToFile(String filePath, ArrayList<Task> tasks) throws IOException {
+        File f = new File(filePath);
+        try {
+            if (!f.exists()) {
+                f.createNewFile();
+            }
+            try (FileWriter writer = new FileWriter(filePath)) {
+                for (Task task : tasks)
+                    if (task != null) {
+                        writer.write(task.toString() + System.lineSeparator());
+                    }
+                writer.close();
+            }
+        } catch (IOException e) {
+            System.out.println("Error writing to file: " + e.getMessage());
+            e.printStackTrace();
+        }
+    }
+
+    private static void readTasksFromFile(String filePath) throws FileNotFoundException {
+        File f = new File(filePath);
+        Scanner s = new Scanner(f);
+        while (s.hasNext()) {
+            String command = s.nextLine();
+            String[] commandWords = savedTask(command);
+            loadTask(commandWords);
+        }
+        s.close();
+    }
+
+    private static String[] savedTask(String savedTask) {
+        return savedTask.split(" ", 2);
+    }
+
+    private static void loadTask(String[] commandWords) {
+        switch (commandWords[0].charAt(3)) {
+        case 'T':
+            addTask(new Todo(commandWords[1]));
+            break;
+        case 'D':
+            String description = commandWords[1].split(" \\(by: ")[0];
+            String by = commandWords[1].split(" \\(by: ")[1].replace(")","").trim();
+            addTask(new Deadline(description, by));
+            break;
+        case 'E':
+            String desc = commandWords[1].split(" \\(from: ")[0];
+            String startTime = commandWords[1].split(" \\(from: ")[1].split(" to: ")[0];
+            String endTime = commandWords[1].split(" \\(from: ")[1].split(" to: ")[1].replace(")", "").trim();
+            addTask(new Event(desc, startTime, endTime));
+            break;
+        default:
+            sendError();
+            break;
+        }
+        if (commandWords[0].charAt(6) == 'X') {
+            markTask(String.valueOf(commandWords[0].charAt(0)));
+        }
+    }
+}
```

Comment:
The main class is doing a lot of heavy lifting and needs serious refactoring/reorganization.
See A-MoreOOP and A-Packages for more instructions